### PR TITLE
[All] py.typed, annotate_with_static

### PR DESCRIPTION
**Pull request by Quarub**

### CODE REVIEW

**Comment by Quarub** [comment]
Sorry, I found something which led me to believe that there's a bug but it seems everything is ok.

**Comment by thetheodor** [comment]
@Quarub the current commit contains a lot of unrelated changes, e.g., adding `annotate_with_static` and packaging related things. 

Could you split this into multiple ones?

File: python_src/dead_instrumenter/instrumenter.py [hunk 1]
```diff
@@ -59,3 +59,38 @@ def instrument_program(
...
+
+    clang_resolved = find_binary(Binary.CLANG) if not clang else str(clang)
+    includes = find_include_paths(clang_resolved, file, flags)
+
+    cmd = [str(instrumenter_resolved), "--mode", "globals", str(file)]
+    for path in includes:
+        cmd.append(f"--extra-arg=-isystem{str(path)}")
+    cmd.append("--")
+
+    subprocess.run(cmd, capture_output=True, check=True)
```
Comment (thetheodor) [inline python_src/dead_instrumenter/instrumenter.py:94]: Should we add logging support to these?

**Comment by thetheodor** [comment]
LGTM

**Pull request merged**